### ISSUE TITLE
AddTitanNode

### ISSUE THREAD

**Issue by berrylands** [Krever/lambda-titan-dynamodb-seed#4]
My first test via Lambda failed because the role did not have CreateTable authority. I added that.

The Second test yields:

{
  "errorMessage": "Could not initialize class example.Titan$",
  "errorType": "java.lang.NoClassDefFoundError",
  "stackTrace": [
    "example.AddNodeHandler.handleRequest(Handlers.scala:51)",
    "example.AddNodeHandler.handleRequest(Handlers.scala:47)"
  ]
}

**Comment by Krever** [comment]
I have just cloned the repo and verified if it still works, and I was able to add and query the titan node. Are you sure you've created the lambda function with *-assembly.jar?

**Comment by berrylands** [comment]
I did, but I cheated.

When I ran 'create-lambda.sh', the lambda functions did not seem to appear. That was my silly mistake - I hadn't noticed that this was being created in the EU region. So I assumed there was something wrong with the script and endeavored to create the Lambda Functions manually.

I have now realised my error, found the functions in the EU region - and am delighted to report that they work! Thank you for going to the trouble of verifying the problem, and also for creating this project!
